lib/docs/filters/core/home_url.rb

```diff
@@ -0,0 +1,16 @@
+module Docs
+  class HomeUrlFilter < Filter
+    def call
+      html.prepend(home_url_html) if home_url
```

Comment:
You'll need to add a class attribute in `doc.rb`, [`here`](https://github.com/Thibaut/devdocs/blob/dfdd4f373ef42a927c97b0da4d5612893d7b0232/lib/docs/core/scraper.rb#L92-L93), and in `filter.rb` to make this work. Maybe add some tests too.